script.js

```diff
@@ -1,12 +1,123 @@
 //You can edit ALL of the code here
+var newEpisodes = [];
+var searchInput = document.getElementById("searchInput");
+var pickEpisode = document.getElementById("select-episode");
+const rootElem = document.getElementById("root");
+var allEpisodes = getAllEpisodes();
+
+function formatEpisode(season, number) {
+    if (season < 10) season = "0" + season;
+    if (number < 10) number = "0" + number;
+    return "S" + season + "E" + number;
+}
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+    rootElem.innerHTML = "";
+    makePageForEpisodes(allEpisodes);
+    searchInput.addEventListener("keyup", liveSearch);
+    makeInputForEpisodes(allEpisodes);
+    pickEpisode.addEventListener("change", pickAnEpisod);
 }
 
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+    for (let i = 0; i < episodeList.length; i++) {
+        //The episode card
+        var episodeElm = document.createElement("div");
+        episodeElm.style.backgroundColor = "white";
+        episodeElm.style.maxWidth = "280px";
+        episodeElm.style.border = "2px solid white ";
+        episodeElm.style.borderRadius = "10px";
+        episodeElm.style.margin = "10px";
+        //episodeElm.style.alignItems = "center";
+        episodeElm.style.marginTop = "0px";
+        rootElem.appendChild(episodeElm);
+        //The Name
+        var episodeName = document.createElement("h3");
+        episodeName.innerText =
+            episodeList[i].name +
+            "-" +
+            formatEpisode(episodeList[i].season, episodeList[i].number);
+        episodeName.style.border = "2px solid black";
+        episodeName.style.borderRadius = "10px";
+        episodeName.style.padding = "20px";
+        episodeName.style.textAlign = "center";
+        episodeName.style.marginTop = "0px";
+        episodeElm.appendChild(episodeName);
+        // The Image
+        var episodeImg = document.createElement("img");
+        episodeImg.src = episodeList[i].image.medium;
+        episodeImg.style.alignSelf = "center";
+        episodeImg.style.margin = "auto";
+        episodeImg.style.display = "block";
+        //episodeImg.style.width = "50%";
+        episodeElm.appendChild(episodeImg);
+        //The Episode Summary
+        var episodesummary = document.createElement("div");
+        episodesummary.innerHTML = episodeList[i].summary;
```

Comment:
Fantastic! You recignised that `summary` already contains HTML elements that have to be turned into HTML tags instead of escaped and displayed as text simple on the site.